apiserver/plane/api/views/issue.py

```
@@ -59,7 +59,7 @@
     IssueTypeCustomProperty,
     Workspace
 )
-from plane.utils.issue_filters import issue_filters
+from plane.utils.issue_filters import issue_filters, build_custom_property_q_objects
```

Pylint (3.3.7)

[error] No name 'build_custom_property_q_objects' in module 'plane.utils.issue_filters'

(E0611)